doc/traits/notifications.rst

```diff
@@ -27,5 +27,9 @@ Example
                    - committers
                    - component_diff_owners
                    - codeowners
+                   slack:
+                        channel_cfgs:
```

Comment:
nit: de-dent by one to match indentation of other lines (4 spaces). Also, `channel_name_s_` expects a list and not a single argument.